harvester/run_harvest.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+python ./src/run_daily_weather.py || {
+    echo "run_daily_weather.py failed";
+    exit 1;
```

Comment:
If you do it like this the second command will not be executed if the first one fails. (I think)

Reply:
I hate bash scripting. changed in last commit.